src/test/java/core/basesyntax/service/impl/FruitTransactionServiceImplTest.java

```diff
@@ -0,0 +1,108 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FruitTransactionService;
+import core.basesyntax.strategy.BalanceOperationHandler;
+import core.basesyntax.strategy.FruitTransactionStrategy;
+import core.basesyntax.strategy.FruitTransactionStrategyImpl;
+import core.basesyntax.strategy.OperationHandler;
+import core.basesyntax.strategy.PurchaseOperationHandler;
+import core.basesyntax.strategy.ReturnOperationHandler;
+import core.basesyntax.strategy.SupplyOperationHandler;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FruitTransactionServiceImplTest {
```

Comment:
```suggestion
public class FruitTransactionServiceTest {
```
the same for other similar cases